Automattic/delphin
# Add svg logo to the header

This pull request changes the logo in the header to use an `svg` file.

<img width="707" alt="screen shot 2016-07-21 at 15 18 34" src="https://cloud.githubusercontent.com/assets/275961/17026005/6da44c3c-4f56-11e6-81af-2a4330173eda.png">
#### Testing instructions
1. Run `git checkout add/svg-logos` and start your server, or open a [live branch](https://delphin.live/?branch=add/svg-logos)
2. Open the [`Home` page](http://delphin.localhost:1337/)
3. Check that the logo at the top looks like the screenshot.
#### Reviews

@Automattic/sdev-feed
